cmd/logger.go

```diff
@@ -106,38 +95,58 @@ func (log *Logger) Printf(format string, args ...interface{}) {
 	}
 }
 
-// NewLogger - returns a new initialized logger.
-func NewLogger() *Logger {
-	logger := logrus.New()
-	logger.Out = ioutil.Discard
-	logger.Level = logrus.DebugLevel
-
-	l := &Logger{
-		logger:        logger,
-		consoleTarget: NewConsoleLogger(),
+func trimTrace(f string) string {
+	trimStrings := []string{
+		pathJoin(GOPATH, "/src/"),
```

Comment:
Note that `GOPATH` is multivalued e.g. `GOPATH=${HOME}/workspace:/shared/golang/gopath` is allowed and valid.

Reply:
@balamurugana ,
This should still be fine.
If `GOPATH` is set to something other than the `{HOME}/workspace`, then the trimming will be ineffective.
Please let me known if you suggest to trim only `{HOME}` and insted of using `GOPATH` and `GOROOT` by leaving `workspace/` and `go/` behind as part of the path.

Reply:
@balamurugana ,

I misunderstood your comment. I discussed it with @harshavardhana and I am looking into it.

Reply:
Done.